.github/workflows/test_on_push.yaml

```diff
@@ -0,0 +1,23 @@
+name: Run tests on any Push event
+
+on:
```

Comment:
Also I'd suggest:
- to extend the list of events with pull-requests
- and to simplify conditions a bit
https://github.com/tsafin/tarantool-merge/blob/206281d3b52035d534d0e5b3fbff185ceaefbaf4/.github/workflows/luarocks-build.yml#L3

Reply:
Thanks! Changed.